### DISCUSSION TITLE
Keras Error

### DISCUSSION THREAD

**Discussion by ELITSZORD** [teamalgoritma/community#686] [Q&A] [answered]
Saya ada kendala ini tidak bisa import keras. kondisinya saya install tensorflow dan keras menggunakan conda akan tetapi sudah saya ganti dengan pip (karena lebih baik menggunakan pip) . Kemudian udh downgradde ke versi tensorflow 2.11 , dan serta sebelumnya udh melakukan  pip install tensorflow-estimator==2.1.*  karena ada error sebelumnya yaitu "partially initialized module 'charset_normalizer` has no attribute md_mypyc  (most likely due to a circular import)". Kira2 solusinya apa ya terkait problem ini?

![keras-error](https://github.com/teamalgoritma/community/assets/114390619/8e554ef8-4ef4-466d-9e65-679e287729df)

Berikut list library saya.

[requirements.txt](https://github.com/teamalgoritma/community/files/11699496/requirements.txt)

**Comment by kinantytasyao** [comment]
Dear @ELITSZORD 

Terima kasih atas pertanyaannya, akan kami coba telusuri ya Pak permasalahannya

**Comment by nadyadtm** [comment, marked as answer]
Dear @ELITSZORD 

Silakan untuk coba mengimport keras dengan cara berikut, contohnya

`from tensorflow.keras.preprocessing import sequence`

Semoga membantu
